src/TaskManager/TaskManager/Monai.Deploy.WorkflowManager.TaskManager.csproj

```diff
@@ -62,12 +62,12 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
+    <PackageReference Include="Monai.Deploy.Messaging" Version="0.1.10" />
+    <PackageReference Include="Monai.Deploy.Messaging.RabbitMQ" Version="0.1.10" />
     <PackageReference Include="Monai.Deploy.Storage" Version="0.2.10" />
-    <PackageReference Include="Monai.Deploy.Messaging" Version="0.1.9" />
     <PackageReference Include="Monai.Deploy.Storage.MinIO" Version="0.2.10" />
-    <PackageReference Include="Monai.Deploy.Messaging.RabbitMQ" Version="0.1.9" />
-    <PackageReference Include="NLog" Version="5.0.5" />
-    <PackageReference Include="NLog.Web.AspNetCore" Version="5.1.5" />
+    <PackageReference Include="NLog" Version="5.0.4" />

```

Comment:
was a downgrade to nlog intentional ?

Reply:
Nope missed it in merge process. Good catch.